Automattic/wp-calypso
# Jetpack Plans: Remove old unused components

#### Proposed Changes

This PR performs some follow-up cleanup after #70428, where old Jetpack plan components that are no longer in use are suggested to be removed.

#### Testing Instructions

* Verify that the removed components are no longer in use.
* Verify Jetpack Cloud and Calypso blue plans/pricing pages still work well as per the test instructions of #70428.
